pennylane/devices/default_qubit.py

```diff
@@ -136,6 +138,18 @@ def adjoint_state_measurements(tape: QuantumTape) -> (Tuple[QuantumTape], Callab
         )
 
     params = tape.get_parameters()
+
+    if device_vjp:
+        for p in params:
+            if (
+                qml.math.requires_grad(p)
+                and qml.math.get_interface(p) == "tensorflow"
+                and "32" in qml.math.get_dtype_name(p)
```

Comment:
dare I ask.. shouldn't we check for `complex64` here?

Reply:
Ah.... yes... yes we do :(